st2api/st2api/controllers/v1/packs.py

```diff
@@ -0,0 +1,35 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2api.controllers.resource import ResourceController
+from st2common.models.api.pack import PackAPI
+from st2common.persistence.pack import Pack
+
+__all__ = [
+    'PacksController'
+]
+
+
+class PacksController(ResourceController):
```

Comment:
Ah good ol' inheritance :P

This class has literally no method and everything is from the base class. Something very strange about that one ... also I am sure the linter must be unhappy seeing no public methods?